ipaserver/install/installutils.py

```diff
@@ -144,10 +154,12 @@ def verify_fqdn(host_name, no_host_dns=False, local_hostname=True):
     :param local_hostname: If true, run additional checks for local hostnames
     """
     if len(host_name.split(".")) < 2 or host_name == "localhost.localdomain":
-        raise BadHostError("Invalid hostname '%s', must be fully-qualified." % host_name)
+        raise BadHostError("Invalid hostname '%s', must be fully-qualified."
```

Comment:
Same here, don't reformat lines that you don't touch.

Reply:
done, i've discard all chages of PEP8 that I was making. fixing just in the lines I've changed.